invenio_communities/assets/semantic-ui/js/invenio_communities/members/components/SearchWithRoleSelection.js

```diff
@@ -0,0 +1,168 @@
+/*
+ * This file is part of Invenio.
+ * Copyright (C) 2024 CERN.
+ *
+ * Invenio is free software; you can redistribute it and/or modify it
+ * under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+import { RadioSelection } from "@js/invenio_communities/members/components/bulk_actions/RadioSelection";
+import { ErrorMessage } from "@js/invenio_communities/members/components/ErrorMessage";
+import { i18next } from "@translations/invenio_communities/i18next";
+import PropTypes from "prop-types";
+import React, { Component } from "react";
+import { Trans } from "react-i18next";
+import { Button, Form, Modal } from "semantic-ui-react";
+import { UsersApi } from "../../api/UsersApi";
+import { MembersSearchBar } from "../invitations/invitationsModal/MemberSearchBar";
+import { SelectedMembers } from "./bulk_actions/SelectedMembers";
+
+export class SearchWithRoleSelection extends Component {
+  constructor(props) {
+    super(props);
+    this.usersApi = new UsersApi();
+    this.state = {
+      role: undefined,
+      selected: {},
+      loading: false,
+      error: undefined,
+    };
+  }
+
+  updateSelected = (members) => {
+    this.setState({ selected: members });
+  };
+
+  addEntityToSelected = (member) => {
+    const { selected } = this.state;
+    this.setState({ selected: { ...selected, ...member } });
+  };
+
+  handleRoleUpdate = (role) => {
+    this.setState({ role: role });
+  };
+
+  handleActionClick = async () => {
+    const { action, onSuccessCallback, message, notify } = this.props;
+    const { selected, role } = this.state;
+    this.setState({ loading: true, error: undefined });
+    try {
+      await action(selected, role, message, notify);
+      this.setState({ loading: false });
+      onSuccessCallback();
+    } catch (error) {
+      this.setState({ loading: false, error: error });
+    }
+  };
+
+  render() {
+    const {
+      roleOptions,
+      modalClose,
+      searchBarTitle,
+      searchBarTooltip,
+      doneButtonText,
+      doneButtonIcon,
+      doneButtonTip,
+      radioLabel,
+      selectedItemsHeader,
+      messageComponent,
+      existingEntities,
+      existingEntitiesDescription,
+    } = this.props;
+    const { selected, loading, error } = this.state;
+    const selectedCount = Object.keys(selected).length;
+
+    return (
+      <>
+        <div className="rel-pl-2 rel-pr-2 rel-pb-2 rel-pt-2">
```

Comment:
if I remember correctly this should do the job:
```suggestion
        <div className="default-padding">
```

Reply:
You are right! Updated